backend/danswer/server/search_backend.py

```diff
@@ -59,10 +59,8 @@ async def promote_admin(
     return
 
 
-@router.get("/direct-qa", response_model=QAResponse)
-def direct_qa(
-    question: QAQuestion = Depends(), user: User = Depends(current_user)
-) -> QAResponse:
+@router.post("/direct-qa")
```

Comment:
might as well make the search endpoints post too? even though none of them log at the moment I guess :P